v1/services/storage/source.go

```diff
@@ -5,30 +5,18 @@ import (
 	"github.com/influxdata/influxdb/v2/kit/platform"
 )
 
-// this is easier than fooling around with .proto files.
```

Comment:
Some notable changes in `v1/services/storage` related to the "source":

- The exported `GetReadSource` from `v1/services/storage/store.go` was not used anywhere, and only the `getReadSource` function was used in the `v1/services/storage/store.go` methods
- I replaced the `getReadSource` function with the `GetReadSource` function, moving `GetReadSource` to here, and updated the references to `getReadSource`. This is helpful because some of the tests need to use `GetReadSource`.
- I also updated the protobuff definition for the `ReadSource` to be in the required form with `OrgID` and `BucketID`